src/components/pages/WritingSub/RenderWritingSub.js

```diff
@@ -36,13 +36,20 @@ export const RenderWritingSub = props => {
         handleOk={() => {
           setModalVisible(false);
         }}
-        instructions={modalText || modalInstructions.writingSub}
+        instructions={modalText}
         style={{ fontSize: '1.5rem' }}
       />
       <div className="writing-sub-container">
         <Row className="main-row">
-          <p>{props.tasks.story.writingPrompt}</p>
+          {/* This will be the topic the user needs to write about. The dummy writingPrompt data in the seeds are just place holders for now and need to get updated */}
```

Comment:
Love the explanatory text for future devs, especially bc that seed data is really confusing!